lib/log/main.go

```diff
@@ -54,9 +65,55 @@ func init() {
 		panic("Unable to create a logger")
 	}
 
-	Log = RawLog.Sugar()
+	Log = sentryInit(RawLog).Sugar()
+}
+
+func sentryInit(log *zap.Logger) *zap.Logger {
+	errConfig := config.ParseEnv()
+	if errConfig != nil {
+		panic("Error parsing environment variables: " + errConfig.Error())
+	}
+	if !config.SentryEnabled() {
+		return log
+	}
+	errSentry := sentry.Init(sentry.ClientOptions{
+		Dsn:              config.SentryDSN(),
+		Environment:      config.SentryEnvironment(),
+		Release:          config.SentryRelease(),
+		TracesSampleRate: 1.0,
+		AttachStacktrace: true,
+	})
+	if errSentry != nil {
+		// this is called before log is initialized
+		fmt.Fprintf(os.Stderr, "Failed to initialize Sentry: %s\n", errSentry)
+		return log
```

Comment:
should this panic?

Reply:
Probably, since it would only reach this is if SentryEnabled==true